test/Base.t.sol

```diff
@@ -45,4 +47,25 @@ contract BaseTest is Defaults, Utils {
         setTokenDecimals(decimals0, decimals1);
         oracle = new ExposedLPOracle(MOCK_POOL, address(helper));
     }
+
+    function setMockOrder() internal {
```

Comment:
Nit: I think this function may want to take real parameters (maybe not now, but at some point). Specifically, token balances and weights so this can be used on different pool classes and reserves.